dev/release/update_change_log.sh

```diff
@@ -50,14 +50,14 @@ OUTPUT_PATH="${PROJECT}/CHANGELOG.md"
 pushd ${SOURCE_TOP_DIR}
 
 # reset content in changelog
-git co "${SINCE_TAG}" "${OUTPUT_PATH}"
+git checkout "${SINCE_TAG}" "${OUTPUT_PATH}"
 # remove license header so github-changelog-generator has a clean base to append
-sed -i '1,18d' "${OUTPUT_PATH}"
+sed -i.bak '1,18d' "${OUTPUT_PATH}"
 
 docker run -it --rm \
     -e CHANGELOG_GITHUB_TOKEN=$CHANGELOG_GITHUB_TOKEN \
     -v "$(pwd)":/usr/local/src/your-app \
-    githubchangeloggenerator/github-changelog-generator:1.16.2 \
+    githubchangeloggenerator/github-changelog-generator \
```

Comment:
use latest

Reply:
might want to pin to the latest version number instead to avoid breaking changes in the future.

Reply:
🤔  https://hub.docker.com/r/githubchangeloggenerator/github-changelog-generator/tags

Seems like `latest` has no other tag (as in `latest` is newer than `1.16.2` but there are no other numbered versions newer than `1.16.2`) 😞 

![Screen Shot 2022-02-11 at 6 00 42 AM](https://user-images.githubusercontent.com/490673/153580452-54ccec0a-47bb-48b2-b5e9-ec7967d58920.png)